pegasus/sites.v3/code.org/public/poetry.md.partial

```diff
@@ -1,37 +0,0 @@
----
-title: Coding with Poetry
-theme: responsive
-set_dir: true
-social:
-  "og:title": "Coding with Poetry"
-  "twitter:title": "Coding with Poetry"
-  "og:description": "Students learn about computer science concepts and writing poetry in this CS Connections module."
-  "twitter:description": "Students learn about computer science concepts and writing poetry in this CS Connections module."
-  "og:image": "https://code.org/images/social-media/poetry-og-image-1200x630.png"
-  "og:image:width": 1200
-  "og:image:height": 630
-  "twitter:card": "summary_large_image"
-  "twitter:image:src": "https://code.org/images/social-media/poetry-og-image-1200x630.png"
```

Comment:
I don't think we need this anymore since /poetry isn't in this file and we don't seem to be using any social media on this page: https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/src/social_metadata.rb

Reply:
This is good to remove 👍